src/Webform.js

```diff
@@ -1777,7 +1777,7 @@ export default class Webform extends NestedDataComponent {
             return;
         }
         const captchaComponent = [];
-        eachComponent(this.components, (component) => {
+        this.eachComponent((component) => {
```

Comment:
this.eachComponent iterates only through direct children, but eachComponent from utils iterates all components tree. This change can break captcha.